twilight-cache-inmemory/src/lib.rs

```diff
@@ -12,6 +12,7 @@
     clippy::unnecessary_wraps
 )]
 
+pub mod interfaces;
```

Comment:
Interfaces aren't a Rust concept, why not just name it "traits"?

Reply:
I realized that after writing it, but I don't like either of the namings. I would probably tend towards naming them `CachableGuild` instead of `GuildInterface` or `GuildTrait` and so on, what do you think about that?